src/cozmo/util.py

```diff
@@ -134,6 +134,15 @@ def degrees(self):
         '''float: The angle in degrees.'''
         return self._radians / math.pi * 180
 
+    @property
+    def abs_value(self):
+        """:class:`cozmo.util.Angle`: The absolute value of the angle.
+        
+        If the Angle is positive then it just returns the Angle, otherwise it
```

Comment:
This comment should probably be updated to reflect that we always call abs now (and therefore this is always a new angle) - e.g.:
"If the Angle is positive then it returns a copy of this Angle, otherwise it returns -Angle."